### PR TITLE
Avoid closing preview panel when file switching

### PR DESCRIPTION
**Pull request by OlivierCavadenti**
When we switch files with ANTLR4 preview panel, it's closed.
Now with this commit, we never close preview panel.
Each grammar have his preview conserved.
I keep it very simple (only remove hide previews calls), say me if we need more complex logic.

Resolve https://github.com/antlr/intellij-plugin-v4/issues/409

### CODE REVIEW

**Comment by parrt** [comment]
Hi!  What happens when you switch away from a grammar file though? Does it still show the preview? I think I remember that causing all sorts of issues because events are sent but we don't have a visible grammar to pull data from.

**Comment by parrt** [comment]
ping @OlivierCavadenti

File: src/main/java/org/antlr/intellij/plugin/ANTLRv4PluginController.java [hunk 1]
```diff
@@ -278,11 +278,9 @@ public void currentEditorFileChangedEvent(VirtualFile oldFile, VirtualFile newFi
 		}
 		if ( newFile.getName().endsWith(".g") ) {
 			LOG.info("currentEditorFileChangedEvent ANTLR 4 cannot handle .g files, only .g4");
-			hidePreview();
```
Comment (parrt) [inline src/main/java/org/antlr/intellij/plugin/ANTLRv4PluginController.java:281]: probably should leave this one in. I can fix.

**Comment by parrt** [comment]
It works well, actually!  I can replicate easilly, but @OlivierCavadenti could you update your commit to use git `-s` to sign?

**Comment by OlivierCavadenti** [comment]
> It works well, actually! I can replicate easilly, but @OlivierCavadenti could you update your commit to use git `-s` to sign?

Hi, I just do it now, sorry for the delay by the way.

**Comment by parrt** [comment]
Hi! I think it's still has an issue with the "signature" on the commit: https://github.com/antlr/intellij-plugin-v4/pull/520/checks?check_run_id=6238810717

**Comment by parrt** [comment]
It's only one line and I can make the change if you want in my fork but I thought it would be good to give you credit for the PR :)

**Comment by parrt** [comment]
awesome. thanks!

**Pull request merged**